src/common/ascii.h

```diff
@@ -492,6 +492,38 @@ struct ascii_logo {
 $C1            ################                      \
 $C1                 #######                          "
 
+#define ASCII_TRANSMETA \
+"$C1                                                 \
+$C1                                                  \
+$C1              ###########                         \
+$C1          ###################                     \
+$C1        ########    ############                  \
+$C1       #####               ########               \
+$C1      ####                    #######             \
+$C1     ####          ###########   ######           \
+$C1     ####         ##############   #####          \
+$C1     ####        ######    #######   ####         \
+$C1      ###        #####        #####   ####        \
+$C1      ####       ######        ####    ####       \
+$C1       ####       ######      #####    ####       \
+$C1        ####        ######    ####     ####       \
+$C1         #####        #######         #####       \
+$C1          #####         ##################        \
+$C1           ######          #############          \
+$C1             ######                               \
+$C1               #######                            \
+$C1                 ########                         \
+$C1                    ########                      \
+$C1                        #########                 \
+$C1                           #####################  \
+$C1                                ################# \
+$C1                                                  "
+
```

Comment:
Please remove the remaining whitespaces, there should be only one whitespace of separation between the end of the logo and the typedef